cmd/bee/cmd/cmd.go

```diff
@@ -42,6 +42,8 @@ const (
 	optionNamePaymentTolerance     = "payment-tolerance"
 	optionNameResolverEndpoints    = "resolver-options"
 	optionNameGatewayMode          = "gateway-mode"
+	optionNameClefSignerEnable     = "clef-signer-enable"
```

Comment:
do we need both flags? maybe infer that `clef-signer-enabled` whenever `clef-signer-endpoint` is defined?

Reply:
To not infer things seems more consistent with how the other "enable"-flags in bee work. E.g. `debug-api-addr`does not infer `debug-api-enable` and `tracing-endpoint` does not infer `tracing-enable` (I think).